clickhouse_connect/driver/asyncclient.py

```diff
@@ -64,11 +64,12 @@ def min_version(self, version_str: str) -> bool:
         """
         return self.client.min_version(version_str)
 
-    def close(self):
+    async def close(self):
```

Comment:
Does this have to be async?  Can we just call the executor.shutdown method synchronously?